### ISSUE TITLE
packages.config parser is missing a "prerelease" attribute

### ISSUE THREAD

**Issue by heldchen** [chocolatey/choco#1382]
### What You Are Seeing?

there is currently no way to specify that a prerelease version of package should be used when installing one or more packages through a `packages.config` file. the xml parser (https://github.com/chocolatey/choco/blob/master/src/chocolatey/infrastructure.app/configuration/PackagesConfigFilePackageSetting.cs) does not support an attribute to allow prereleases.

using the `--pre` cli parameter is not an option when a package.config file contains more than one `<package>` and one would only needs some of the packages to use prerelease versions.

### What is Expected?

one should be able to specify that a prerelease for a package should be used to install, for example like so:
```
<?xml version="1.0" encoding="utf-8"?>
<packages>
  <package id="git" />
  <package id="javaruntime" />
  <package id="phpstorm" prerelease="true" />
</packages>
```

with such a file, the version `2017.2.2-EAP` (most current approved prerelease) should be installed instead of `2017.2.1` (most current approved release):
``` 
C:\>choco info phpstorm -r
phpstorm|2017.2.1

C:\>choco info phpstorm -r --pre
phpstorm|2017.2.2-EAP

```

**Comment by ferventcoder** [comment]
You are correct - a good workaround for now is to specify the version of the prerelease in the config file.

**Comment by ferventcoder** [comment]
#886 is similar (this may even be a dupe of that).

**Comment by byronsanchez** [comment]
Just ran into this issue myself, so a +1 from me :)

**Comment by andreasblueher** [comment]
+1 from my side aswell
